test/jdk/java/awt/Desktop/8064934/bug8064934.java

```diff
@@ -32,23 +32,16 @@
  * @build jdk.test.lib.Platform
  * @run main bug8064934
  */
-import jdk.test.lib.Platform;
+
 import java.awt.*;
 import java.io.File;
 import java.io.IOException;
-import java.security.AccessController;
-import java.security.PrivilegedAction;
 
 public class bug8064934 {
     private static final String NO_ASSOCIATION_ERROR_MESSAGE = "Error message: No application is associated with" +
             " the specified file for this operation.";
 
     public static void main(String[] args) {
-        // This test is intended only for Windows
-        if (!AccessController.doPrivileged((PrivilegedAction<Boolean>) Platform::isWindows)) {
-            System.out.println("The test is for Windows platform only");
-            return;
-        }
```

Comment:
Since this test has @requires (os.family == "windows") tag, os check is redundant, thus removed.